docs/reference/commandline/secret_create.md

```diff
@@ -28,7 +28,7 @@ Options:
 ## Description
 
 Creates a secret using standard input or from a file for the secret content. You must run this
-command on a manager node.
+command on a manager node. The content of secret must be less than 500 KB.
```

Comment:
I can feel some kind of improper adding this in the first paragraph. How about the updated format? @thaJeztah